generatorcore/industry2018/energy_demand.py

```diff
@@ -0,0 +1,390 @@
+            "Fact_I_P_metal_steel_secondary_ratio_fec_to_prodvol_2018",
+            lambda x: div(1, x),
+        ),
+        "Fact_I_P_metal_steel_secondary_ratio_CO2e_pb_to_prodvol": replace_value(
+            "Fact_I_P_metal_steel_secondary_ratio_CO2e_pb_to_prodvol_2018"
+        ),
+        "Fact_I_P_metal_steel_secondary_ratio_CO2e_cb_to_prodvol": replace_value(
+            "Fact_I_P_metal_steel_secondary_ratio_CO2e_eb_to_prodvol_2018"
+        ),
+        "Fact_I_P_metal_fec_pct_of_nonfe": replace_value(
+            "Fact_I_P_metal_fec_pct_of_nonfe_2018"
+        ),
+        "Fact_I_P_metal_nonfe_ratio_prodvol_to_fec": replace_value(
+            "Fact_I_P_metal_nonfe_ratio_fec_to_prodvol_2018", lambda x: div(1, x)
+        ),
+        "Fact_I_P_metal_nonfe_ratio_CO2e_pb_to_prodvol": replace_value(
+            "Fact_I_P_metal_nonfe_ratio_CO2e_pb_to_prodvol_2018"
+        ),
+        "Fact_I_P_metal_nonfe_ratio_CO2e_cb_to_prodvol": replace_value(
+            "Fact_I_P_metal_nonfe_ratio_CO2e_cb_to_prodvol_2018"
+        ),
+        # sontige Industrie Fakten
+        "Fact_I_P_other_fec_pct_of_paper": replace_value(
+            "Fact_I_P_other_fec_pct_of_paper_2018"
+        ),
+        "Fact_I_P_other_paper_ratio_prodvol_to_fec": replace_value(
+            "Fact_I_P_other_paper_ratio_fec_to_prodvol_2018", lambda x: div(1, x)
+        ),
+        "Fact_I_P_other_paper_ratio_CO2e_pb_to_prodvol": replace_value(
+            "Fact_I_P_other_paper_ratio_CO2e_pb_to_prodvol_2018"
+        ),
+        "Fact_I_P_other_paper_ratio_CO2e_cb_to_prodvol": replace_value(
+            "Fact_I_P_other_paper_ratio_CO2e_cb_to_prodvol_2018"
+        ),
+        "Fact_I_P_other_fec_pct_of_food": replace_value(
+            "Fact_I_P_other_fec_pct_of_food_2018"
+        ),
+        "Fact_I_P_other_food_ratio_prodvol_to_fec": replace_value(
+            "Fact_I_P_other_food_ratio_fec_to_prodvol_2018", lambda x: div(1, x)
+        ),
+        "Fact_I_P_other_food_ratio_CO2e_pb_to_prodvol": replace_value(
+            "Fact_I_P_other_food_ratio_CO2e_pb_to_prodvol_2018"
+        ),
+        "Fact_I_P_other_food_ratio_CO2e_cb_to_prodvol": replace_value(
+            "Fact_I_P_other_food_ratio_CO2e_cb_to_prodvol_2018"
+        ),
+        "Fact_I_P_other_fec_pct_of_further": replace_value(
+            "Fact_I_P_other_fec_pct_of_further_2018"
+        ),
+        "Fact_I_P_other_further_prodvol": replace_value(
+            "Fact_I_P_other_further_prodvol_2018"
+        ),
+        "Fact_I_P_other_further_ratio_CO2e_pb_to_fec": replace_value(
+            "Fact_I_P_other_2d_ratio_CO2e_pb_to_fec_2018"
+        ),
+        "Fact_I_P_other_further_ratio_CO2e_cb_to_fec": replace_value(
+            "Fact_I_P_other_further_ratio_CO2e_cb_to_fec_2018"
+        ),
+        "Fact_I_P_other_2efgh_ratio_CO2e_pb_to_fec": replace_value(
+            "Fact_I_P_other_2efgh_ratio_CO2e_pb_to_fec_2018"
+        ),
+    }
+
+    new_facts.append_rows(additional_facts)
```

Comment:
Also prinzipiell ist der Industry Branch jetzt fertig. Was allerdings noch zu tun ist, ist das Renaming der Fakten, was ich jetzt bis jetzt in einer Funktion zur runtime (siehe hier) mache, anstelle dieser Funktion einfach in unserer facts.csv im Datenrepo ändere.

Reply:
Hmm. I don't really understand the plan here yet.

Reply:
Leider sieht man in dieser Ansicht in der Conversation Section nicht wirklich, was ich hier gemacht habe. Ich habe (fast) alle Fakten, die im Industriebereich gebraucht werden in einen Standard umbenannt und neu in unseren Fakten DataFrame[] hinzugefügt (Die Fakten mit den jetzt nicht mehr benutzen Namen sind also erstmal noch drin).

Das habe ich aber nur gemacht, damit ich beim Testen nicht ständig das neu das Datenrepo und den jeweiligen Hash anpassen muss, damit ich auf meine umbenannten Fakten zugreifen kann. Jetzt, wo alles läuft, würde ich die Fakten in der .csv umbennen (und umrechnen, das manchmal 1/"fact" verwendet wird) und dann alles, was mit der funktion 
```
def prepare_facts(inputs: Inputs):
```
zu tun hat wieder löschen.